astroquery/gaia/core.py

```diff
@@ -228,22 +235,25 @@ def load_data(self, ids, *, data_release=None, data_structure='INDIVIDUAL',
 
         output_file_specified = False
         if output_file is None:
-            output_file = os.path.join(os.getcwd(), temp_dirname, downloadname_formated)
+            output_file = os.path.join(os.getcwd(), temp_dirname,
+                                       downloadname_formated)
```

Comment:
We do let longer lines in astroquery (`max-line-length = 120`). It is supposed to be picked up from the config files, but I'm happy to add new sections that your editor/IDE would respect.

Reply:
Sorry for the mistake, I misconfigured my pycharm IDE: I set the  max line length to 79 characters. I have reverted the formatting changes